var/spack/repos/builtin/packages/py-tensorflow-estimator/package.py

```diff
@@ -31,23 +32,26 @@ class PyTensorflowEstimator(Package):
 
     extends("python")
 
-    depends_on("python@3.7:", type=("build", "run"), when="@2.9.0")
-
-    depends_on("py-keras@2.9.0:2.9", type=("build", "run"), when="@2.9.0")
-    depends_on("py-keras@2.8.0:2.8", type=("build", "run"), when="@2.8.0")
-    depends_on("py-keras@2.7.0:2.7", type=("build", "run"), when="@2.7.0")
-    depends_on("py-keras@2.6.0:2.6", type=("build", "run"), when="@2.6.0")
-    depends_on("py-tensorflow@2.9.0:2.9", type=("build", "run"), when="@2.9.0")
-    depends_on("py-tensorflow@2.8.0:2.8", type=("build", "run"), when="@2.8.0")
-    depends_on("py-tensorflow@2.7.0:2.7", type=("build", "run"), when="@2.7.0")
-    depends_on("py-tensorflow@2.6.0:2.6", type=("build", "run"), when="@2.6.0")
-    depends_on("py-tensorflow@2.5.0:2.5", type=("build", "run"), when="@2.5.0")
-    depends_on("py-tensorflow@2.4.0:2.4", type=("build", "run"), when="@2.4.0")
-    depends_on("py-tensorflow@2.3.0:2.3", type=("build", "run"), when="@2.3.0")
-    depends_on("py-tensorflow@2.2.0:2.2", type=("build", "run"), when="@2.2.0")
-    depends_on("py-tensorflow@2.1.0:2.1", type=("build", "run"), when="@2.1")
-    depends_on("py-tensorflow@2.0.0:2.0", type=("build", "run"), when="@2.0.0")
-    depends_on("py-tensorflow@1.13.1", type=("build", "run"), when="@1.13.0")
+    depends_on("python@3.7:", when="@2.9:", type=("build", "run"))
+
+    for ver in ["2.10", "2.9", "2.8", "2.7", "2.6"]:
+        depends_on("py-keras@" + ver, when="@" + ver, type=("build", "run"))
+
+    for ver in [
+        "2.10",
+        "2.9",
+        "2.8",
+        "2.7",
+        "2.6",
+        "2.5",
+        "2.4",
+        "2.3",
+        "2.2",
+        "2.1",
+        "2.0",
+        "1.13",
```

Comment:
This doesn't match the original requirement of `py-tensorflow@1.13.1`, does that matter?

Reply:
Probably not, all TF estimator releases seem to correspond to a single TF release version. It's possible that older versions were more flexible as to which versions they claimed to support, but my experience with TF has been that versions need to match exactly, otherwise bad things happen. No one should be using such an old version of TF anyway.